### PR TITLE
POC - setup orm

### PR DESCRIPTION
**Pull request by dhmoon91**
# Befor putting PR (put x in bracket if you ran it.);

- [ ] Run `pylint $(git ls-files '*.py')` passes

# Overview
https://github.com/dhmoon91/porobot/issues/29

- Adding migration module and DB ORM setup. 
    - Added `summoners` `team_members` and `channels` table to start off.
    - Updated `rank` command to save data into `summoners` table for simple example
- Update readme.

### CODE REVIEW

File: db/models/summoners.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    def __init__(
+        self, channel_id, region, puuid, tier_division, tier_rank, solo_win, solo_loss
+    ):
+        self.channel_id = channel_id
+        self.region = region
+        self.puuid = puuid
+        self.tier_division = tier_division
+        self.tier_rank = tier_rank
+        self.solo_win = solo_win
+        self.solo_loss = solo_loss
```
Comment (jung1411) [inline db/models/summoners.py:46]: 여기에 lp points (0~100lp) 집어넣는게 좋을듯.

File: db/models/channel.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, String, DateTime
+from ..db import Base
+import datetime
+
+
+class Channel(Base):
+    __tablename__ = "channels"
+
+    channe_id = Column(Integer, primary_key=True)
```
Comment (jung1411) [inline db/models/channel.py]: typo on channel_id

File: db/models/summoners.py (hunk 1 above)
Comment (jung1411) [reply db/models/summoners.py:46]: 나중에 primary_position, secondary_position 집어넣을수도 있고

File: README.md [hunk 3]
```diff
@@ -36,6 +36,42 @@ pre-commit install --hook-type pre-push
...
+   * To apply migration;  
+   `alembic upgrade head`
+
+   * **Make sure you update the models reference in db/models/ to represent newly added schema**
+
+   * To revert one migration;  
+   `alembic downgrade -1`
+
+   * To revert all migration;  
+   `alembic downgrade base`
```
Comment (dhmoon91) [inline README.md:74]: Updated readme for db

File: .pylintrc [hunk 4]
```diff
@@ -0,0 +1,618 @@
...
+        deprecated-sys-function,
+        exception-escape,
+        comprehension-escape,
+        too-few-public-methods,
+        too-many-arguments,
+        fixme,
+        too-many-instance-attributes,
+        too-many-branches,
+        too-many-statements,
+        too-many-locals,
```
Comment (dhmoon91) [inline .pylintrc:152]: Disabled; 
```
too-few-public-methods,
too-many-arguments,
fixme,
too-many-instance-attributes,
too-many-branches,
too-many-statements,
too-many-locals,


```

**Review by jung1411 — APPROVED** [review]

**Pull request merged**